src/animation/AnimationAction.js

```diff
@@ -70,78 +69,77 @@ THREE.AnimationAction._new =
 	this.weight = 1;
 	this._effectiveWeight = 1;
 
-	this.repetitions = Infinity; 		// no. of repetitions when looping
+	this.repetitions = Infinity; // no. of repetitions when looping
 
-	this.paused = false;				// false -> zero effective time scale
-	this.enabled = true;				// true -> zero effective weight
+	this.paused = false; // false -> zero effective time scale
+	this.enabled = true; // true -> zero effective weight
 
-	this.clampWhenFinished 	= false;	// keep feeding the last frame?
+	this.clampWhenFinished = false; // keep feeding the last frame?
 
-	this.zeroSlopeAtStart 	= true;		// for smooth interpolation w/o separate
-	this.zeroSlopeAtEnd		= true;		// clips for start, loop and end
+	this.zeroSlopeAtStart = true; // for smooth interpolation w/o separate
+	this.zeroSlopeAtEnd = true; // clips for start, loop and end
 
 };
 
 THREE.AnimationAction._new.prototype = {
-
 	constructor: THREE.AnimationAction._new,
 
 	// State & Scheduling
 
-	play: function() {
+	play: function () {
 
 		this._mixer._activateAction( this );
 
 		return this;
 
 	},
 
-	stop: function() {
+	stop: function () {
 
 		this._mixer._deactivateAction( this );
 
 		return this.reset();
 
 	},
 
-	reset: function() {
+	reset: function () {
 
 		this.paused = false;
 		this.enabled = true;
 
-		this.time = 0;			// restart clip
-		this._loopCount = -1;	// forget previous loops
-		this._startTime = null;	// forget scheduling
```

Comment:
All these comments were neatly aligned, and now they read worse than before. (Keep in mind the tab sized used for display on github is 8).